MAUI/Cartesian-Charts/SplineRangeArea.md

```diff
@@ -0,0 +1,135 @@
+
+N> The Cartesian chart has [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) as its default content.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+
+    <chart:SfCartesianChart.XAxes>
+        <chart:CategoryAxis/>
+    </chart:SfCartesianChart.XAxes>
+
+    <chart:SfCartesianChart.YAxes>
+        <chart:NumericalAxis/>
+    </chart:SfCartesianChart.YAxes>   
+
+    <chart:SplineRangeAreaSeries ItemsSource="{Binding Data}"
+                                 XBindingPath="XValue"
+                                 High="HighValue"
+                                 Low="LowValue"/>
+
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+CategoryAxis primaryAxis = new CategoryAxis();
+chart.XAxes.Add(primaryAxis);
+
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.YAxes.Add(secondaryAxis);
+
+SplineRangeAreaSeries series = new SplineRangeAreaSeries()
+{
+    ItemsSource = new ViewModel().Data,
+    XBindingPath = "XValue",
+    High="HighValue",
+    Low="LowValue",
+};
+
+chart.Series.Add(series);
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Spline range area chart type in MAUI Chart]()
+
+### Spline rendering types
+
+The [Type]() property allows to change the rendering type of spline curve in series. The default value of [Type]() is [Natural](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SplineType.html#Syncfusion_Maui_Charts_SplineType_Natural).
+
+The following types are used in [SplineRangeAreaSeries]():
+
+* `Natural`
+* `Monotonic`
+* `Cardinal`
+* `Clamped`
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCartesianChart>
+
+        <chart:SfCartesianChart.XAxes>
+            <chart:CategoryAxis/>
+        </chart:SfCartesianChart.XAxes>
+
+        <chart:SfCartesianChart.YAxes>
+            <chart:NumericalAxis/>
+        </chart:SfCartesianChart.YAxes>  
+
+        <chart:SplineRangeAreaSeries ItemsSource="{Binding Data}"
+                                     XBindingPath="XValue"
+                                     High="HighValue"
+                                     Low="LowValue"
+                                     Type="Cardinal"/>
+
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+CategoryAxis primaryAxis = new CategoryAxis();
+chart.XAxes.Add(primaryAxis);
+
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.YAxes.Add(secondaryAxis);
+
+SplineRangeAreaSeries series = new SplineRangeAreaSeries()
+{
+    ItemsSource = new ViewModel().Data,
+    XBindingPath = "XValue",
+    High="HighValue",
+    Low="LowValue",
+    Type = SplineType.Cardinal
+};
+
+chart.Series.Add(series);
+
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Spline types in MAUI Chart]()
```

Comment:
@EzhilarasanElangovan31 update as  - Spline types in MAUI Spline range area chart Chart

Reply:
Updated